docs/examples/google-cloud-function.ts

```diff
@@ -0,0 +1,24 @@
+import { Telegraf } from 'telegraf'
+
+const { BOT_TOKEN, FUNCTION_NAME, PROJECT_ID, REGION } = process.env
+
+if (BOT_TOKEN === undefined) {
```

Comment:
Maybe even check for falsy = empty strings?
```suggestion
if (!BOT_TOKEN) {
```

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md

Reply:
Might be helpful as we dont have to know about this falsy stuff then.

Reply:
Telegraf does the "sloppy" check internally anyway:

https://github.com/telegraf/telegraf/blob/66a0020e8522c81d600f2c309800aad058cacbd5/src/core/network/client.ts#L336-L341

It's a leftover from JS days, possibly still useful to people writing JS.